src/Symfony/Component/Form/Resources/translations/validators.en.xlf

```diff
@@ -18,6 +18,110 @@
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Please enter a valid date.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Please select a valid file.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The form is invalid.</source>
+                <target>The form is invalid.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>The hidden field is invalid.</source>
+                <target>The hidden field is invalid.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please enter an integer.</source>
+                <target>Please enter an integer.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid language.</source>
+                <target>Please select a valid language.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please select a valid locale.</source>
+                <target>Please select a valid locale.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a valid money amount.</source>
+                <target>Please enter a valid money amount.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>Please enter a number.</source>
+                <target>Please enter a number.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>The password is invalid.</source>
+                <target>The password is invalid.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>Please enter a percentage value.</source>
+                <target>Please enter a percentage value.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>The values do not match.</source>
+                <target>The values do not match.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please enter a valid time.</source>
+                <target>Please enter a valid time.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please select a valid timezone.</source>
+                <target>Please select a valid timezone.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid URL.</source>
+                <target>Please enter a valid URL.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>The search button is invalid.</source>
+                <target>The search button is invalid.</target>
```

Comment:
I don't have any better suggestion, but why does this message mentions a button? A search input just is a raw text input.

Reply:
Maybe something like "Please enter a valid search term."?

Reply:
👍

Reply:
done